backend-kotlin/src/test/kotlin/com/example/backendkotlin/usecase/CreateVillageUseCaseUT.kt

```diff
@@ -25,79 +32,166 @@ import java.util.UUID
 class CreateVillageUseCaseUT(
     @MockkBean
     private val villageRepository: VillageRepository,
+
+    @MockkBean
+    private val userRepository: UserRepository,
+
+    @MockkBean
+    private val rUserVillageRepository: RUserVillageRepository,
 ) : DescribeSpec() {
     @InjectMockKs
     private lateinit var target: CreateVillageUseCase
 
     override fun afterTest(f: suspend (Tuple2<TestCase, TestResult>) -> Unit) {
         // テスト後にMockの挙動を初期化する
-        confirmVerified(villageRepository)
+        confirmVerified(villageRepository, userRepository, rUserVillageRepository)
```

Comment:
細かテクだけど、一行単位で改行して trailing comma を使うほうが今後の git 差分が減ったり
```suggestion
        confirmVerified(
        villageRepository,
        userRepository,
        rUserVillageRepository,
        )
```

Reply:
d3033ad